lambdaclass/curse_of_myrra
# In-game UI refactor

## Changes
- In game UI
- Character selection UI
- Delete sprites and optimize the way we handle them
## How to test it
- In character selection go through each player and their skills to check they are working as expected (selected and not selected styles)
-   Use all skill inputs and check their cooldown and disable status is working
